backend/app/api/schemas.py

```diff
@@ -0,0 +1,61 @@
+from datetime import datetime
+from enum import Enum
+from typing import Optional
+
+from pydantic import BaseModel
+
+
+class Filter(str, Enum):
+    free = 'free'
+
+
+class Sender(str, Enum):
+    volunteer = 'volunteer'
+    client = 'client'
+
+
+class MessageBase(BaseModel):
+    created_at: datetime
+    content: str
+    sender: Optional[Sender]  # TODO: add recognize service, fields must be not optional
+
+
+class MessageCreate(MessageBase):
+    pass
+
+
+class Message(MessageBase):
+    pass
+
+
+class SupportBase(BaseModel):
+    question: str
+    client_id: Optional[int]  # TODO: add recognize service, fields must be not optional
+    volunteer_id: Optional[int]
+    messages: Optional[list[Message]] = []
+
+    class Config:
+        orm_mode = True
+
+
+class SupportCreate(SupportBase):
+    id: Optional[int]
```

Comment:
I don't see a case when `id` should be provided here. Do we ever create a support thread with id already?

Reply:
Moreover, at the moment of creation we neither have a `volunteer_id`, nor `messages` (they are always default), do we?
(those come from `SupportBase`)